docs/touchablewithoutfeedback.md

```diff
@@ -23,6 +21,56 @@ function MyComponent(props) {
 </TouchableWithoutFeedback>;
 ```
 
+### Example
+
+```SnackPlayer name=TouchableWithoutFeedback
+
+import React, { useState } from "react";
+import { StyleSheet, TouchableWithoutFeedback, Text, View } from "react-native";
+
+export default function TouchableWithoutFeedbackExample() {
+  const [count, setCount] = useState(0);
+
+  const onPress = () => {
+    setCount(count + 1);
+  };
+
+  return (
+    <View style={styles.container}>
+      <TouchableWithoutFeedback onPress={onPress}>
+        <View style={styles.button}>
+          <Text> Touch Here </Text>
+        </View>
+      </TouchableWithoutFeedback>
+
+      <View style={styles.countContainer}>
+        <Text style={styles.countText}>{count !== 0 ? count : null}</Text>
```

Comment:
Just a question: What is wrong with rendering `0`?  😃

It will be nice if you can move counter above button and add "Count:" label like in [TouchableOpacity](https://reactnative.dev/docs/next/touchableopacity) Snack example.

Reply:
Thank you for the review!
That's easier to understand, I'll fix it.

Reply:
thanks！
Fixed it